dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -1801,16 +1783,19 @@ public void deleteWorkflow(@ApiParam(hidden = true) @Parameter(hidden = true, na
         }
     }
 
-    /**
-     * Finds the tool by Id, and then checks that it exists and that the user has access to it
-     * @param entryId Id of tool of interest
-     * @param user User to authenticate
-     * @return Tool
-     */
-    private Workflow findWorkflowByIdAndCheckWorkflowAndUser(Long entryId, User user) {
-        Workflow workflow = workflowDAO.findById(entryId);
-        checkEntry(workflow);
-        checkUser(user, workflow);
-        return workflow;
+    @POST
+    @Path("/github/release")
+    @Timed
+    @Consumes(MediaType.APPLICATION_FORM_URLENCODED)
+    @UnitOfWork
+    @RolesAllowed({ "curator", "admin" })
```

Comment:
`@Operation` has a nested `@ApiResponse` that can describe response codes. For that matter, these should described in OpenAPI